app-frontend/src/app/components/histogram/nodeHistogram/nodeHistogram.controller.js

```diff
@@ -302,6 +313,30 @@ export default class NodeHistogramController {
         }
     }
 
+    recalculateBreakpointsFromRange(min, max) {
+        let oldmin = _.first(this._breakpoints).value;
+        let oldmax = _.last(this._breakpoints).value;
+
+        let currentRange = oldmax - oldmin;
+        let newRange = max - min;
+        if (this._breakpoints && currentRange !== newRange && newRange > 0) {
```

Comment:
I'm not sure that `currentRange !== newRange` will always give the correct results; e.g. what if `oldMax` = 90, `oldMin` = 0, and `newMax` = 100, `newMin` = 10? It's possible this won't be a problem though depending on when this will be getting called.

Reply:
`recalculateBreakpointsFromRange` is only ever called with either the min or max changing.
I need to work on naming